episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+In the USA and Canada, no deaths were attributed to the virus in the first ten days following the World Health Organization's declaration of a public health emergency. Even under similar circumstances at the early stage of the global pandemic, public health officials, policymakers and the general public want to know the virulence of an emerging infectious agent.
+
+[Fraser et al., 2009](https://www.science.org/doi/full/10.1126/science.1176062) reinterpreted the data assessing the biases and getting a clinical severity lower than the 1918 influenza pandemic but comparable with that seen in the 1957 pandemic.
+
+::::::::::::::::::::
+
+:::::::::::::::::::: discussion
+
+Based on your experience:
+
+- Share any previous outbreak in which you participated in its response.
+
+Answer to these questions:
+
+- How did you assess the clinical severity of the outbreak?
+- What were the primary sources of bias?
+- What did you do to take into account the identified bias?
+- What complementary analysis would you do to solve the bias?
+
+:::::::::::::::::::: 
+
+## Delay-adjusted CFR
+
+Nishiura et al., 2009 developed a method that considers the time delay from the onset of symptoms to death.
+
+Real-time outbreaks may have a number of deaths that are insufficient to determine the time distribution between onset and death. Therefore, we can estimate the _distribution delay_ from historical outbreaks or reuse the ones available in systematic reviews from repositories like `{epiparameter}` or `{epireview}`.
+
+Let's use `{epiparameter}`:
+
+```{r, message=FALSE, warning=FALSE}
+# Get delay distribution
+onset_to_death_ebola <-
+  epiparameter::epidist_db(
+    disease = "Ebola",
+    epi_dist = "onset_to_death",
+    single_epidist = TRUE
+  )
+```
+
+To calculate the delay-adjusted CFR, we can use the `cfr_static()` function with the `data` and `delay_density` arguments.
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
+
+:::::::::::::::::: callout
+
+### Use the `<epidist>` class
+
+When using an `<epidist>` class object we can use this expression as a template `function(x) density(<EPIDIST_OBJECT>, x)`
+
+For distribution functions with parameters not available in `{epiparameter}`, we suggest you two alternatives: 
+
+- Create an `<epidist>` class object, to plug into other R packages of the outbreak analytics pipeline. Read the [reference documentation of `epiparameter::epidist()`](https://epiverse-trace.github.io/epiparameter/reference/epidist.html), or
+
+- Read `{cfr}` vignette for [a primer on working with delay distributions](https://epiverse-trace.github.io/cfr/articles/delay_distributions.html).
```

Comment:
Somewhere here it would also be good to add a caveat about using continuous distributions in place of discrete ones, and when this is okay. See the [delay distributions vignette in {cfr}](https://epiverse-trace.github.io/cfr/articles/delay_distributions.html#using-continuous-and-discrete-distributions) for an explanation.

Reply:
We could add a spoiler callout for this. Is discretizing an <epidist> object an appropriate alternative to pass discrete distributions?

```
:::::::::::::::::: spoiler

## When to use discrete distributions?

For `cfr_*()`, the most appropriate choice to pass are **discrete** distributions. This is because we will work with daily case and death data.

We can assume that evaluating the Probability Distribution Function (PDF) of a *continuous* distribution is equivalent to the Probability Mass Function (PMF) of the equivalent *discrete* distribution.

However, this assumption may not be appropriate for distributions with greater picks. For example, when onset-to-death is strongly peaked with a low variance. In such cases, the average disparity between the PDF and PMF is expected to be more pronounced compared to distributions with broader spreads.

In a scenario like the last one, you can opt to discretise the continuous distribution using `epiparameter::discretise()` to an `<epidist>` object.

::::::::::::::::::
```

Reply:
Yes, seems like a good idea.